src/python/pants/bin/local_pants_runner.py

```diff
@@ -174,11 +176,19 @@ def create(
         :param daemon_graph_session: The graph helper for this session.
         """
         build_root = get_buildroot()
-
         global_options = options_bootstrapper.bootstrap_options.for_global_scope()
         # This works as expected due to the encapsulated_logger in DaemonPantsRunner and
         # we don't have to gate logging setup anymore.
-        setup_logging_from_options(global_options)
+
+        native = Native()
```

Comment:
We use `Native()` elsewhere in the file and mutate the value there. Sanity check: this is safe to use `Native()` twice because it is a singleton, correct?

Reply:
yes `Native` is a singleton